test/playwright/visual.spec.js

```diff
@@ -26,6 +26,12 @@ test.beforeEach(async ({context}) => {
 });
 
 test('visual', async ({page, extensionId}) => {
+    // Open welcome page
+    await page.goto(`chrome-extension://${extensionId}/welcome.html`);
+
```

Comment:
There should probably be a `toBeVisible` check added here for some element otherwise it's possible the page might not be loaded yet (in theory)

Reply:
Added one for the green `All Yomitan features are available.` text